src/GithubToken.elm

```diff
@@ -0,0 +1,6 @@
+module GithubToken exposing (..)
+
+
+token : String
+token =
+    "Get your Github toke from here: https://github.com/settings/tokens"
```

Comment:
Looks good, I should have documented this, my bad. However, the token needs to be of the form:

```
token : String
token =
    "token GITHUB_TOKEN_HERE"
```

Do you mind updating the README?

Additionally we could also update the code so `token` is a `Maybe String`, and then it can be set to `Nothing` by default. What do you think? The token is only really important on the actual deployed site. Let me know if you fancy doing that!